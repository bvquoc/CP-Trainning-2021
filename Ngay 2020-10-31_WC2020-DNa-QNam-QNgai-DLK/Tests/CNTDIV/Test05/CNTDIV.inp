5
639
623
890
713
677
