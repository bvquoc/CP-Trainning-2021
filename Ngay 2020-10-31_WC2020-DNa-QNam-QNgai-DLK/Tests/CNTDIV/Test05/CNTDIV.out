821
2881
2232
9330
301
